### PR TITLE
Prepare release candidate 1.0.0rc1

### PR DESCRIPTION
**Pull request by notoraptor**
@nouiz @lamblin

### CODE REVIEW

File: NEWS_DEV.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ Highlights:
  - Theano now internally uses ``sha256`` instead of ``md5`` to work on systems that forbide ``md5`` for security reason
  - Removed old GPU backend ``theano.sandbox.cuda``. New backend ``theano.gpuarray`` is now the official GPU backend
  - Support more debuggers for ``PdbBreakpoint``
+ - Make sure MKL uses GNU OpenMP
```
Comment (nouiz) [inline NEWS_DEV.txt]: I would add as a subpoint more details:

matrix dot product (gemm) with numpy 1.13 and mkl from conda could is some cases return wrong results. We have reported the problem upstream and have a work around to have an error raised with information how to fix it to the user.

File: NEWS.txt [hunk 2]
```diff
@@ -12,6 +12,11 @@ We recommend that every developer updates to this version.
 
 Highlights:
  - Make sure MKL uses GNU OpenMP
+
+   - **NB**: Matrix dot product (``gemm``) with numpy ``1.13`` and ``mkl`` from conda
```
Comment (nouiz) [inline NEWS.txt]: I would do:
Matrix dot product (``gemm``) with numpy ``1.13`` and ``mkl`` from conda
->
Matrix dot product (``gemm``) with and ``mkl 2018`` from conda

I think this is independent of numpy and depend only on the mkl version. I just tried to install something and it tried to install numpy 1.13 with mkl 2017 in conda. 

@abergeron does that sound good to you?

**Comment by notoraptor** [comment]
@nouiz Updated.

File: NEWS.txt (hunk 2 above)
Comment (Gam3356) [reply NEWS.txt]: Willie Parish IPHONE 12 Pro max

**Pull request merged**